### PR TITLE
preact x support 👀

### PR DESCRIPTION
**Pull request by ForsakenHarmony**

### CODE REVIEW

File: packages/cli/lib/lib/babel-config.js [hunk 1]
```diff
@@ -24,7 +24,10 @@ module.exports = function(env, options = {}) {
 			require.resolve('@babel/plugin-transform-react-constant-elements'),
 			isProd &&
 				require.resolve('babel-plugin-transform-react-remove-prop-types'),
-			[require.resolve('@babel/plugin-transform-react-jsx'), { pragma: 'h' }],
+			[
+				require.resolve('@babel/plugin-transform-react-jsx'),
+				{ pragma: 'h', pragmaFrag: 'Fragment' },
```
Comment (reznord) [inline packages/cli/lib/lib/babel-config.js]: what is the `pragmaFrag`?

**Comment by prateekbh** [comment]
Do we have PreactX in 3.0 now!!!??

**Comment by ForsakenHarmony** [comment]
when it gets released 🙃 

this was just fixes I had to make to get it to work

File: packages/cli/lib/lib/webpack/webpack-base-config.js [hunk 2]
```diff
@@ -236,6 +238,7 @@ module.exports = function(env) {
 			}),
 			new webpack.ProvidePlugin({
 				h: ['preact', 'h'],
+				Fragment: ['preact', 'Fragment'],
```
Comment (developit) [inline packages/cli/lib/lib/webpack/webpack-base-config.js]: awesome, I was wondering how we'd automate this. woot!

**Pull request merged**